walletsetup.go

```diff
@@ -382,6 +392,7 @@ func promptConsoleSeed(reader *bufio.Reader) ([]byte, error) {
 			}
 
 			fmt.Printf("Invalid seed specified.  Must be a "+
+				"33 word seed using the PGP wordlist or "+
```

Comment:
Other sized seeds encoded as a wordlist should also be valid, just as they are for hex.